documentation/modules/adding-source-provider.adoc

```diff
@@ -21,35 +21,150 @@ You can add {a-rhv} source provider by using the {ocp} web console.
 
 * {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate
 endif::[]
+ifdef::ova[]
+= Adding an Open Virtual Appliance (OVA) source provider
+
+You can add an Open Virtual Appliance (OVA) source provider that originated from or is comatible with VMware vSphere by using the {ocp} web console.
```

Comment:
```suggestion
You can add an Open Virtual Appliance (OVA) source provider that originated from or is compatible with VMware vSphere by using the {ocp} web console.
```
besides the typo, I think we should phrase it a bit differently: we support imports of OVAs that were exported from vSphere and we may be able to import OVAs that are compatible with vSphere
as [the virt-v2v guys wrote](https://libguestfs.org/virt-v2v-input-vmware.1.html): "With this method you must first export the guest (eg. from vSphere) as an .ova file, which virt-v2v can then read directly. Note this method only works with files exported from VMware, not OVA files that come from other hypervisors or management systems, since OVA is only a pretend standard and is not compatible or interoperable between vendors."
and indeed we had issue in the past with OVA files that were exported from other systems and were supposedly compatible with vSphere

Reply:
Suggested rephrasing: "You can add an Open Virtual Appliance (OVA) source provider that was imported from VMware vSphere by using the {ocp} web console. It is possible to use an OVA source provider that is _compatible_ with vSphere but the migration might fail. "